webrender/res/cs_clip_rectangle.fs.glsl

```diff
@@ -2,35 +2,51 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+void clip_against_ellipse_if_needed(vec2 pos,
+                                    inout float current_distance,
```

Comment:
given that there is only one output parameter, it would look cleaner (more FP style) if it was returned directly instead of `inout`